### ISSUE TITLE
Overmind looks down on tyrant

### ISSUE THREAD

**Issue by necessarily-equal** [Unvanquished/Unvanquished#1340]
![Screenshot from 2021-03-01 13-41-39](https://user-images.githubusercontent.com/59283660/109507533-23909a00-7a9f-11eb-8704-22a6fab1f39c.png)

The tyrant isn't supposed to be *that* small.

I suspect the origin of the model is off

**Comment by Pretuer** [comment]
Overmind is a little shy my friend

**Comment by bmorel** [comment]
> The tyrant isn't supposed to be that small.

Overmind simply does not looks at the head of aliens. Size of tyrant:

```
mins              -32 -32 -22
maxs              32 32 70
```

This means if OM looks at origin which is what I would guess, then it looks rants at ~25% of rant's size.
Tyrant is the only alien with that kind of proportions, others being between 50% and 40%.

What would be the other consequences of changing the mins/maxs values of tyrant, here?

**Comment by necessarily-equal** [comment]
Maybe overmind should look at the head of the creature instead. I think this would solve it well?

**Comment by bmorel** [comment]
indeed, using `dir = target.position + target.z` would be the simpler fix.

**Comment by bmorel** [comment]
in cg_buildable:

```
void CG_Buildable( centity_t *cent )
{
  ...
          // Get absolute angles to target.
        VectorSubtract( cg_entities[ es->otherEntityNum ].lerpOrigin, eyeOrigin, dirToTarget );
        VectorNormalize( dirToTarget );
        vectoangles( dirToTarget, angles );
}
```
